### PR TITLE
DWPF-1723 Update Navigation Page Layout

### PR DESCRIPTION
**Pull request by alsyx**
- [x] JIRA ticket referenced in title
- [x] Title is clear and concise
- [ ] Description gives any relevant detail
- [ ] Tests are up to date
- [ ] Documentation is up to date

### CODE REVIEW

File: src/content/templates/content/navigation_page.html [hunk 1]
```diff
@@ -1,22 +1,22 @@
-{% extends "1_col_content.html" %}
+{% extends "2_col_content.html" %}
 {% load wagtailcore_tags %}
 
 {% block content %}
     <div class="dwds">
-        {% include 'feedback.html' %}
         <h1>{{ page.title }}</h1>
-        <div class="ws-content-grid">
+        <div class= "grid">
```
Comment (marcelkornblum) [inline src/content/templates/content/navigation_page.html:7]:
```suggestion
        <div class="grid">
```

File: src/content/templates/content/navigation_page.html [hunk 2]
```diff
@@ -1,22 +1,22 @@
...
-        {% for block in page.body %}
-            {% include_block block %}
+    </div>
+{% endblock %}
+
+{% block tertiary_col_content %}
+    <div class="dwds stack">
+        {% include 'feedback.html' %}
+        {% for block in page.secondary_elements %}
+            <div>{% include_block block %}</div>
```
Comment (marcelkornblum) [inline src/content/templates/content/navigation_page.html:19]: we shouldn't need extra `<div>`s here - do we need them?

Same issue as above

File: src/content/templates/content/navigation_page.html [hunk 3]
```diff
@@ -1,22 +1,22 @@
...
 {% load wagtailcore_tags %}
 
 {% block content %}
     <div class="dwds">
-        {% include 'feedback.html' %}
         <h1>{{ page.title }}</h1>
-        <div class="ws-content-grid">
+        <div class= "grid">
             {% for block in page.primary_elements %}
                 <div>{% include_block block %}</div>
```
Comment (marcelkornblum) [inline src/content/templates/content/navigation_page.html:9]:
```suggestion
                {% include_block block %}
```

File: src/dw_design_system/dwds/styles/variables.scss [hunk 4]
```diff
@@ -38,4 +40,5 @@
   --font-lineheight-body: 1.5rem;
   --font-lineheight-headings: 1.25rem;
   --font-lineheight-interface: 1rem;
+  --font-lineheight-headings: 1.6rem;
```
Comment (marcelkornblum) [inline src/dw_design_system/dwds/styles/variables.scss:43]:
```suggestion
  --font-lineheight-headings: 1.6rem;
  --font-lineheight-interface: 1rem;
```

File: src/content/templates/content/navigation_page.html (hunk 2 above)
Comment (alsyx) [reply src/content/templates/content/navigation_page.html]: After I removes the `div`s, the height of the element gets flexible too. 
Screenshot 2024-05-21 at 10 09 03
so we don't have to add any more code to achieve that @marcelkornblum

Comment (alsyx) [reply src/content/templates/content/navigation_page.html]: + Changed the `font-family`  to be Roboto Flex as well, and the `font-weight` for bold is 600 now.

Comment (marcelkornblum) [reply src/content/templates/content/navigation_page.html]: oh OK then - fair enough !

File: src/dw_design_system/dwds/styles/variables.scss [hunk 5]
```diff
@@ -28,14 +28,16 @@
   --s3: calc(var(--s2) * var(--ratio));
   --s4: calc(var(--s3) * var(--ratio));
   --s5: calc(var(--s4) * var(--ratio));
+  --border-width-card: 2px;
+  --border-top-width-card: var(--s-2);
 
   // Typography
   --measure: 70ch;
-  --font-family-primary: "Roboto", sans-serif;
+  --font-family-primary: "Roboto Flex", sans-serif;
```
Comment (marcelkornblum) [inline src/dw_design_system/dwds/styles/variables.scss:36]: You can't change just this on its own, you also need to customise the import

**Review by marcelkornblum — APPROVED** [review]

**Pull request merged**